siFromSpl.py

```diff
@@ -88,7 +88,12 @@ def si_from_spl( splFile, startFile ):
         #FIXME Need to ensure we are reading the right thing more rigorously
         # Since \x87 is a valid thing in the file check the 5th byte before the one we found
         # marks the start of an si card
-        endLoc = data.find(b'\x87', endLoc + 1)
+
+        # e.g. x80-id-x84||x81-ccardid-x87-firstname-x-lastname-x-club-x97
```

Comment:
I maybe don't remember the format well enough, is there really just an x between firstname and lastname at times.
Also x8C seems unusual there.

Reply:
Ah sorry no that was me just making a note to myself there. Between firstname and lastname there is either a) an x88 (if two separate fields) or b) a 20 (i.e. space) if firstname and lastname were concatenated into a single field. The surname seems to either be end-delimited by x89 or x8c.

For x8c example see 64520.spl